examples/resources/genesyscloud_integration/resource.tf

```diff
@@ -0,0 +1,20 @@
+resource "genesyscloud_integration" "integration1" {
+  name             = "test_integration"
+  intended_state   = "DISABLED"
+  integration_type = "embedded-client-app"
+  config {
+    name = "Premium Client Application Example"
```

Comment:
They are the same thing, same attribute. Yeah I also thought this is a little strange when I was testing it. But in the API, there are two names, one in integration and one in integration config, and I just kept it this way, to sync with API settings. Maybe I can get rid of the config name and just use the integration name?

Reply:
If they're the same attribute in the backend, we should only expose one.

Reply:
okay, I'll change them. 
Just curious, the integration name, not the config name, actually doesn't do anything except gets updated every time after read. So even users modify that name, it won't do anything. The only way to update the name attribute is through the config name. So basically that integration name attr is a read-only attribute. Does that also fit in this situation and considered as an exposed attribute?

Reply:
IMO this is just a case of a confusing API, and since we are a layer above the API we have the ability to deviate from them when they are confusing. In order to make it simpler, we should have one attribute for "name" and when you change it in the config, the name on the backend (and therefore the UI) also changes. If you want to keep the name in the config object that is fine too, just as long as it exists in one place and allows modifying the name.

Reply:
Removed the top-level "name" attribute. Now it only has the one in the integration config.